compiler/rustc_session/src/config.rs

```diff
@@ -1400,7 +1400,10 @@ The default is {DEFAULT_EDITION} and the latest stable edition is {LATEST_STABLE
 pub fn rustc_short_optgroups() -> Vec<RustcOptGroup> {
     vec![
         opt::flag_s("h", "help", "Display this message"),
-        opt::multi_s("", "cfg", "Configure the compilation environment", "SPEC"),
+        opt::multi_s("", "cfg", "Configure the compilation environment and use conditional compilation. SPEC support
+                             [target_arch|target-feature|target_os|target_family|target_env|
+                             target_endian|target_pointer_width|target_vendor|target_has_atomic|
+                             test|debug_assertions|proc_macro|panic] arguments", "SPEC"),
```

Comment:
this isn't really correct. you've listed the *pre-defined* cfgs, which actually in practice *shouldn't* be overridden on the command line (see https://github.com/rust-lang/compiler-team/issues/610). Instead you should give a short description of the possible *syntax* for the cfgs, so people have an idea of how to pass e.g. `feature = "foo"` on the command line.